README.md

```diff
@@ -37,3 +37,6 @@ dependencies) you may also find it useful to add the typed dependencies to
     -   id: mypy
         additional_dependencies: [tokenize-rt==3.2.0]
 ```
+
+Note that if you don't specify additional dependencies, because of the isolated
+virtualenv you might get different output when running `mypy` into your own virtualenv.
```

Comment:
isn't this the same as what lines 31-33 say?

Reply:
Yeah you are right, probably my fault for not giving that enough importance! 👍